src/applications/pensions/labels.jsx

```diff
@@ -8,3 +8,9 @@ export const marriageTypeLabels = {
     'In a civil or religious ceremony with an officiant who signed my marriage license',
   other: 'Some other way',
 };
+
+export const separationTypeLabels = {
+  death: "Spouse's Death",
```

Comment:
this text doesn't match the previous text. It should be `'Spouse’s death'` not `"Spouse's Death"`. I dont know if the capitalization of `Death` matters, but the apostrophe mark used is different. After changing that, I was able to get the cypress tests to pass locally.